recipes/mutmap/meta.yaml

```diff
@@ -24,7 +24,9 @@ requirements:
     - python >=3.5
     - samtools >=1.7
     - bcftools >=1.7
+    - openssl =1.0
```

Comment:
Please remove `=1.0` as that's handled internally by the conda-forge-pinning package.

Reply:
@bioconda/core
I removed "openssl >=1.0".